migration/1696842672748-add_glo_stable_coin_token.ts

```diff
@@ -0,0 +1,103 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { NETWORK_IDS } from '../src/provider';
+import { Token } from '../src/entities/token';
+
+const newTokens = [
+  {
+    name: 'GLO',
```

Comment:
We standardize our naming scheme to `Glo Dollar` for the name and `USDGLO` for the symbol. See https://coinmarketcap.com/currencies/glo-dollar/ for an example. Would be great if we can adopt this here too :-)